src/plugins/commander/commands/member.js

```diff
@@ -0,0 +1,45 @@
+const Command = require('../structs/Command.js');
+const got = require('got');
+
+class MemberCommand extends Command {
+    constructor(bot) {
+        super(bot);
+        this.aliases = ['member', 'verify'];
+
+        this.shortdesc = `Gives you the member role.`;
+        this.desc = `Gives you the member role if don't already have it, requires one edit on dev.`;
+        this.usages = [
+            '!member wiki-username'
+        ];
+    }
+
+    call(message, content) {
```

Comment:
Missing async here to use await below, shame on you soph

Reply:
I didn't add async because I wasn't sure if it needed it (I only added it to got because I thought got needed it to work)